packages/manager/apps/pci-instances/src/data/hooks/instance/selectors/instances.selector.ts

```diff
@@ -72,7 +72,7 @@ const getActionHrefByName = (
   { region, id }: Pick<TAggregatedInstance, 'id' | 'region'>,
 ): TAggregatedInstanceAction['link'] => {
   if (name === 'details') {
-    return { path: id, isExternal: false };
+    return { path: `region/${region}/instance/${id}`, isExternal: false };
```

Comment:
Please, as we made for action urls, use legacy routes because we need to handle hybridation ;)